### PR TITLE
Update create-a-user-mapping-file-for-data-content-migration.md

### PR DESCRIPTION
**Pull request by dariomws306**
https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/1403

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/create-a-user-mapping-file-for-data-content-migration.md [hunk 1]
```diff
@@ -61,12 +61,22 @@ The following example uses Excel to create the CSV file.
...
+
+2.  Click on **Start your first Migration**.
+
+3.  Choose your migration type.
+
+4.  Enter your source information and then click **Next**.
+
+5.  Enter your destination information and then click **Next**.
+
+6.  Review your Migration details and then click **Next**.
```
Comment (JohanFreelancer9) [inline migration/create-a-user-mapping-file-for-data-content-migration.md]:
```suggestion
6.  Review your migration details and then click **Next**.
```

File: migration/create-a-user-mapping-file-for-data-content-migration.md [hunk 2]
```diff
@@ -61,12 +61,22 @@ The following example uses Excel to create the CSV file.
...
+
+7.  On the choose your settings screen expand **View all settings**.
```
Comment (JohanFreelancer9) [inline migration/create-a-user-mapping-file-for-data-content-migration.md]:
```suggestion
7.  On the Choose your settings screen, expand **View all settings**.
```

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**